services/121-service/src/payments/fsp-integration/intersolve-visa/services/visaStatusMap.csv

```diff
@@ -26,6 +35,15 @@
 "TRUE";"ACTIVE";"CARD_STOLEN";"TRUE";"Blocked";"Visa marked card as stolen";"Issue New Card"
 "TRUE";"ACTIVE";"CARD_CLOSED";"TRUE";"Blocked";"Visa marked card as closed";"Issue New Card"
 "TRUE";"ACTIVE";"CARD_EXPIRED";"TRUE";"Blocked";"Card has expired";"Issue New Card"
+"TRUE";"SUBSTITUTED";"CARD_OK";"TRUE";"Paused";"Card has been substituted for a new card";"Unpause, Issue New Card"
+"TRUE";"SUBSTITUTED";"CARD_BLOCKED";"TRUE";"Blocked";"Card has been substituted for a new card";"Issue New Card"
+"TRUE";"SUBSTITUTED";"SUSPECTED_FRAUD";"TRUE";"Suspected Fraud";"Card has been substituted for a new card";"Issue New Card"
```

Comment:
In theory the current wallet should never have the substituted status.. Should we stil map it?